modules/core/keeper/keeper.go

```diff
@@ -36,8 +37,16 @@ func NewKeeper(
 	stakingKeeper clienttypes.StakingKeeper, upgradeKeeper clienttypes.UpgradeKeeper,
 	scopedKeeper capabilitykeeper.ScopedKeeper,
 ) *Keeper {
+	// register paramSpace at top level keeper
+	// set KeyTable if it has not already been set
+	if !paramSpace.HasKeyTable() {
+		keyTable := clienttypes.ParamKeyTable()
+		keyTable.RegisterParamSet(&connectiontypes.Params{})
```

Comment:
Registers client and connection params

Reply:
should this if be removed from 02-client?

Reply:
No if client keeper is registered seperately (e.g. for testing) this will make sure params are still registered.